README.md

```diff
@@ -81,6 +82,11 @@ Install [Foundry](https://book.getfoundry.sh/getting-started/installation).
 
 Run `forge test`.
 
+## Solidity version
+
+The main contracts (`PreLiquidation` and `PreLiquidationFactory`) are written in Solidity 0.8.27.
```

Comment:
To be absolutely clear, the programs are written with Solidity and compiled using `solc` version 0.8.27. In particular, the languages includes a directive (`pragma`) to ensure a given compiler version is used. I don't know if the language specification version is the same as the compiler. Comment is optional.

Reply:
I think the language specification version is the same as the compiler (for instance the custom error in require is specific to Solidity 0.8.27)

Reply:
I'm interested, do you have a link for the language specification that states the version?

Reply:
Not sure what you mean exactly by language specification but you can get "language features" in each solidity release version (for example [here](https://github.com/ethereum/solidity/releases/tag/v0.8.27) for 0.8.27)

Reply:
As mater of fact, when you follow your link you'll realize that the version is attached to the compiler and not the language which are two different objects.